.github/workflows/merge-to-main.yaml

```diff
@@ -0,0 +1,37 @@
+name: Merge to main
+
+on:
+  pull_request:
+    types: [labeled]
```

Comment:
Judging by this and the JS file, we're looking just for SOT approvals and nothing else, right? Meaning: no more 'Ready for Main' label required. Is there any use-case where we would need to look out for the 'do not merge' label? I can't think of some particular use-cases right now, but I just saw the label on a [PR](https://github.com/adobecom/milo/pull/2362) of yours which gave me the idea.

Reply:
Yes, just the SOT approvals & a PR needs to be mergeable.
- Conflicts = PR can't be merged
- Requesting changes = PR can't be merged
- only 3/5 SOT labels = PR will not be merged

I thought about the `do not merge` label, but the standard GitHub workflow is requesting changes on a PR, which should be good enough

Reply:
`Ready for Main` was used to quickly filter/indicate the Stage->Main PRs, but as we have #milo-changelog + a recurring title, that should be good enough to find the prod PRs IMO